src/checkout-buttons/strategies/braintree/braintree-paypal-button-strategy.ts

```diff
@@ -114,6 +127,18 @@ export default class BraintreePaypalButtonStrategy implements CheckoutButtonStra
         return Promise.resolve();
     }
 
+    private renderMessages(amount: number, container: string) {
+        const { paypal } = this._window;
+        if (!paypal || !paypal.Messages) {
+            throw new PaymentMethodClientUnavailableError();
```

Comment:
Does smart payment buttons will be rendered in case of exception here? What will be shown in browser for such case?

Reply:
You are right, better to avoid throwing error to proceed with normal flow

Reply:
Fixed